test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -72,6 +72,21 @@ test.requestHooks(mockEnrollAuthenticatorPassword)(`should navigate to password
   await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
 });
 
+test.requestHooks(mockEnrollAuthenticatorPassword).only(`select password challenge page and hit switch authenticator and re-select password`, async t => {
+  const selectFactorPage = await setup(t);
+  await t.expect(selectFactorPage.getFormTitle()).eql('Set up Authenticators');
+
+  selectFactorPage.selectFactorByIndex(0);
+  const enrollPasswordPage = new FactorEnrollPasswordPageObject(t);
+  await t.expect(enrollPasswordPage.passwordFieldExists()).eql(true);
+  await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
+  await enrollPasswordPage.clickGoBackLink();
+  // re-select password
+  selectFactorPage.selectFactorByIndex(0);
+  await t.expect(enrollPasswordPage.passwordFieldExists()).eql(true);
+  await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
```

Comment:
verify Requests loggers. In this case would expect
- one request for render authenticator list
- one request for render password challenge
- one request for render password challenge again